src/pages/jaewonyang/Main/MainJaeWon.js

```diff
@@ -1,7 +1,120 @@
-import React from "react";
+import React, { useState } from 'react';
+import AsideJaeWon from './Aside/AsideJaeWon';
+import './MainJaeWon.scss';
 
-function Main() {
-  return <div>Main</div>;
+function MainJaeWon() {
+  const [commentInput, setCommentInput] = useState('');
+  const [commentList, setCommentList] = useState([]);
+
+  const handleCommentInput = event => {
+    setCommentInput(event.target.value);
+  };
+
+  const onClick = event => {
+    event.preventDefault();
+    if (commentInput !== '') {
+      let newArr = [...commentList];
+      newArr.push({ account: 'Lily', text: commentInput });
+      setCommentList(newArr);
+    }
+    setCommentInput('');
+  };
+
+  return (
+    <body className="MainJaeWon">
+      <div className="main">
+        <div className="main__left">
+          <header class="main__left__header">
```

Comment:
```suggestion
    <body className="MainJaeWon">
      <div className="main">
        <div className="main__left">
          <header class="main__left__header">
```
body태그는 필용벗을것 같고 클래스명도 살짝 헷갈리지않나,, 싶네요

Reply:
동의합니다아....